Cargo.toml

```diff
@@ -35,6 +35,10 @@ assert_approx_eq = "1.1.0"
 trybuild = "1.0.23"
 rustversion = "1.0"
 
+[build-dependencies]
+walkdir = "~2.3"
+regex = "~1.3"
```

Comment:
I think let's make these dependencies optional with a `cross-compile` feature please 👍

Reply:
I've applied the feature. However I believe this will be useful when targeting one's own system (on unix) instead of running the python script. Although not all the information is present.

Reply:
no longer necessary